StationeersLaunchPad/LaunchPadConfig.cs

```diff
@@ -574,17 +583,38 @@ public static void SaveConfig()
         });
       }
 
-      config.SaveXml(WorkshopMenu.ConfigPath);
+      if (!config.SaveXml(WorkshopMenu.ConfigPath))
+        throw new Exception($"failed to save {WorkshopMenu.ConfigPath}");
     }
 
     private async static UniTask LoadMods()
     {
+      ElapsedStopwatch.Restart();
       LoadState = LoadState.ModsLoading;
 
-      var strategy = new LinearLoadStrategy();
-      await strategy.Load();
+      switch (LoadStrategyType) {
+        default:
+        case LoadStrategyType.Linear: {
+          switch (LoadStrategyMode) {
```

Comment:
more secret c# features for you: you can do a switch expression here to shorten this
```cs
LoadStrategy loadStrategy = (LoadStrategyType, LoadStrategyMode) switch
{
  (LoadStrategyType.Linear, LoadStrategyMode.Serial) => new LoadStrategyLinearSerial(),
  (LoadStrategyType.Linear, LoadStrategyMode.Parallel) => new LoadStrategyLinearParallel(),
  _ => throw new Exception($"invalid load strategy ({LoadStrategyType}, {LoadStrategyMode})")
};
await loadStrategy.LoadMods();
```